snapcraft.yaml

```diff
@@ -0,0 +1,113 @@
+name: desmume-emulator
+version: 0.9.13
+summary: DeSmuME is a Nintendo DS Emulator
+description: |
+  An open-source Nintendo DS Emulators.
+adopt-info: desmume-emulator
+grade: stable
+confinement: strict
+base: core22
+compression: lzo
+parts:
+  desmume-emulator:
+    source: https://github.com/TASEmulators/desmume/archive/refs/tags/release_0_9_13.tar.gz
+    plugin: nil
+    override-build: |
+      LDFLAGS="" CFLAGS="" FFLAGS="" FCFLAGS="" DEB_CFLAGS_SET="" DEB_CXXFLAGS_SET="" DEB_FFLAGS_SET="" DEB_FCFLAGS_SET="" DEB_LDFLAGS_SET="" meson setup --prefix=${CRAFT_PART_INSTALL}/usr desmume/src/frontend/posix/build desmume/src/frontend/posix
+      LDFLAGS="" CFLAGS="" FFLAGS="" FCFLAGS="" DEB_CFLAGS_SET="" DEB_CXXFLAGS_SET="" DEB_FFLAGS_SET="" DEB_FCFLAGS_SET="" DEB_LDFLAGS_SET="" ninja -C desmume/src/frontend/posix/build
+      LDFLAGS="" CFLAGS="" FFLAGS="" FCFLAGS="" DEB_CFLAGS_SET="" DEB_CXXFLAGS_SET="" DEB_FFLAGS_SET="" DEB_FCFLAGS_SET="" DEB_LDFLAGS_SET="" ninja install -C desmume/src/frontend/posix/build
+
+    stage-packages:
+      - libasound2 
+      - libc6 
+      - libcairo2 
+      - libgcc-s1
+      - libgcc-s1      
+      - libgdk-pixbuf-2.0-0 
+      - libgl1
+      - libglade2-0 
+      - libglib2.0-0 
+      - libglu1-mesa
+      - libgtk2.0-0 
+      - libgtkglext1
+      - libosmesa6 
+      - libpango-1.0-0 
+      - libsdl1.2debian 
+      - libsoundtouch1 
+      - libstdc++6 
+      - libtinyxml2.6.2v5
+      - zlib1g
+      - g++
+      - gcc
+      - gettext
+      - make
+      - cmake
+      - nasm
+      - pkg-config
+      - git
+      - zip
+      - build-essential
+      - intltool
+      - libglade2-dev
+      - libgtk2.0-dev
+      - libsdl1.2-dev
+      - libosmesa6-dev
+      - libagg-dev
+      - intltool
+      - quilt
+      - libgtkglext1-dev
+      - libtinyxml-dev
+      - libsoundtouch-dev
+      - dh-autoreconf
+      - meson
+      - libsdl2-dev
+      - libpcap-dev
+      - libgtk-3-dev
+    build-packages:
+      - g++
+      - gcc
+      - gettext
+      - make
```

Comment:
seems to me that one of build-packages vs stage-packages are packages needed for the build vs packages that need to be shipped together with the app.
you only need cmake, make, nasm, gcc, etc in the former category, not in both.

Reply:
exact and I also need to see how to replace the dev with their standard package

Reply:
and I believe that the patches from the rpmfusion team of Fedora are precisely for the architectures which were released in failure of construction so I will see to add them

https://github.com/rpmfusion/desmume/blob/master/desmume-0.9.13-aarch64.patch

https://github.com/rpmfusion/desmume/blob/master/desmume-0.9.13-arm.patch

https://github.com/rpmfusion/desmume/blob/master/desmume-0.9.13-formatstring.patch

https://github.com/rpmfusion/desmume/blob/master/desmume-0.9.13-ppc64le.patch

https://github.com/rpmfusion/desmume/blob/master/desmume-0.9.13-tinyxml.patch